### PR TITLE
feat(rad-1437): change category score to bigdecimal

### PR DESCRIPTION
**Pull request by FrontSide**
as we round to 2 decimal places

### CODE REVIEW

**Comment by fcibecchini** [comment]
why was it removed?

File: aylien/v1/news/api-docs.yaml [hunk 1]
```diff
@@ -5886,8 +5886,10 @@ components:
           nullable: true
         score:
           description: The score of the category
-          format: double
           type: number
+          format: bigdecimal
+          maximum: 1
+          minimum: 0
           nullable: true
```
Comment (fcibecchini) [inline aylien/v1/news/api-docs.yaml:5893]: remove nullable if it's no longer the case

**Review by marcinaylien — DISMISSED** [review]

File: aylien/v1/news/api-docs.yaml (hunk 1 above)
Comment (FrontSide) [reply aylien/v1/news/api-docs.yaml]: removed

**Comment by FrontSide** [comment]
> why was it removed?

The confidence score you mean? I'm assuming because of a products requirement.

**Review by fcibecchini — APPROVED** [review]

**Pull request merged**